.github/workflows/push.yml

```diff
@@ -19,6 +19,8 @@ jobs:
         -D clippy::cast-sign-loss
         -D clippy::checked-conversions
         -A clippy::upper-case-acronyms
+        -A clippy::result-large-err
+        -A special_module_name
```

Comment:
Shall we create an issue not to forget to solve these?

Reply:
Oh, I already resolved those in this branch: https://github.com/tmpolaczyk/witnet-rust/tree/rust-1-65

but I just cherry-picked the first commit because it is trivial to review and it doesn't block the release. The other changes may need some comments.